### ISSUE TITLE
unresolved reference 'google' after pip install google-cloud-translate on PyCharm IDE

### ISSUE THREAD

**Issue by wobeng** [GoogleCloudPlatform/google-cloud-python#3503]
I get an unresolved reference 'google' when I do

```
from google.cloud import translate

client = translate.Client()
```
I'm using a virtual environment. Python 3.6

pip freeze:
cachetools==2.0.0
google-auth==1.0.1
google-auth-httplib2==0.0.2
google-cloud-core==0.24.1
google-cloud-translate==0.24.0
googleapis-common-protos==1.5.2
httplib2==0.10.3
protobuf==3.3.0
pyasn1==0.2.3
pyasn1-modules==0.0.9
rsa==3.4.2
six==1.10.0

![error](https://user-images.githubusercontent.com/11966846/27159396-b23d069a-513a-11e7-9a1b-bd5c3f5e2896.png)

**Comment by dhermes** [comment]
@wobeng This is not uncommon, we've seen it many times. It's because `google` and `google.cloud` are namespace packages and those act a bit differently than normal packages.

How did you install / what else have you installed? This is definitely caused by a "bad" install (either from pre-existing packages "polluting" the namespace or from an install gone wrong).

I'm going to close this issue, but am happy to keep chatting with you here until we can figure out how your install got broken.

**Comment by wobeng** [comment]
@dhermes 

It seems to be a bug on pycharm

https://youtrack.jetbrains.com/issue/PY-2813#u=1444657668935.  The fix is going to be included in 2017.2. 2017.1 is the latest

Thanks

**Comment by dhermes** [comment]
Thanks for the update
